src/library/scala/collection/concurrent/TrieMap.scala

```diff
@@ -912,6 +913,18 @@ extends scala.collection.concurrent.Map[K, V]
       }
     }
   }
+  def fail[K, B](self: TrieMap[K, V]): Option[V] = {
```

Comment:
Oh, that was me just hand-inlining some code to add to the bug report. Wasn't intended to be included here, will remove.